src/main/java/org/jetbrains/bsp/bazel/server/bsp/resolvers/TargetsResolver.java

```diff
@@ -1,41 +1,175 @@
 package org.jetbrains.bsp.bazel.server.bsp.resolvers;
 
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier;
+import com.google.common.collect.ImmutableList;
 import com.google.devtools.build.lib.query2.proto.proto2api.Build;
+import java.security.InvalidParameterException;
```

Comment:
Are you sure this is the exception you wanted?

Reply:
`IllegalStateException` maybe?